.github/workflows/check-tsc-vote.yml

```diff
@@ -0,0 +1,13 @@
+name: Required Reviews
+on:
+  pull_request: {}
+  pull_request_review: {}
```

Comment:
I think you can also restrict it to changes in `kedro-datasets/kedro_datasets` - https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-files-changed-in-a-pull-request

Reply:
Done!